bibliopixel/remote/server.py

```diff
@@ -0,0 +1,93 @@
+from flask import Flask, jsonify, send_from_directory
+from werkzeug.serving import run_simple
+import threading
+import queue
+import os
+import logging
+from .. import log
+
+# I want to keep this to supress the "Running on" message
+# But then it also swallows 404 erros and such in the console :(
+# werk_log = logging.getLogger('werkzeug')
+# werk_log.setLevel(logging.ERROR)
+
+
+def fail(msg='Error'):
+    return {
+        'status': False,
+        'msg': msg,
+        'data': None
+    }
+
+
+def success(data=None, msg='OK'):
+    return {
+        'status': True,
+        'msg': msg,
+        'data': data
+    }
+
+
+class RemoteServer():
```

Comment:
Where have you been? New-style classes are implicit in Python 3!!
I couldn't done:
```
class RemoteServer:
    def __init__(self): pass
```
And it would work they same as (object) or ().